.stylelintrc.json

```diff
@@ -14,7 +12,7 @@
     },
     "no-eol-whitespace": true,
     "function-parentheses-space-inside": "never-single-line",
-    "max-nesting-depth": 6,
+    "max-nesting-depth": 5,
```

Comment:
Why reduced?

Reply:
why not? Goal should be to reduce it even further